frontends/web/src/routes/exchange/components/exchange-providers.tsx

```diff
@@ -47,6 +47,10 @@ const PaymentMethod = ({ methodName }: TPaymentMethodProps) => {
         {t('buy.exchange.creditCard')}
       </span>
     );
+  case 'sofort':
+  case 'bancontact':
+    // hide these payment methods in the overview
+    return '';
```

Comment:
This is not really robust, imo. If the only available deals in a specific situation were the hidden ones, this would show a component without available options. It would be better to have a "hidden" field in the deal struct that can be set in the backend and properly checked in the frontend.

Reply:
I don't know sofort or bancontact (not available in CH), aren't these some kind  of fiat banktransfers anyway?

Hidden field sounds more robust, but I am not 100% confident about how to do the backend changes... I'll dm you later.

Reply:
as discussed I'll leave in ths PR, and we can add isHidden in a new PR